### PR TITLE
fix: visual testing on menu is back

### PR DESCRIPTION
**Pull request by vincentaudebert**
## Type

- Enhancement

#### What is expected?

Fix visual testing for Menu

#### The following changes where made:

(Describe what you did)

1. change userEvent by fireEvent

### CODE REVIEW

**Comment by vincentaudebert** [comment]
Could be a quick fix as ideally we should use userEvent to reproduce real user interaction with button...

**Comment by vincentaudebert** [comment]
Now targeting "body" so we get to screenshot portals properly and completely.

Had to regen all screenshots. No difference except padded white border.

Reactivated all Menu play functions for a working visual testing.

Also raised a bit chromeTolerance from 0.02 to 0.05 to avoid false positive

**Review by baptisteArno — COMMENTED** [review]
It's not well documented on loki's documentation but it seems `chromeTolerance` at the first level of the config object won't have any effect?

See: https://github.com/oblador/loki/issues/113#issuecomment-1181232630

It should be

```
"pixelmatch": {
    "tolerance": 0.00024
  }
```

**Comment by vincentaudebert** [comment]
@baptisteArno Had to dig into the code to make sure. It seems in config file it comes from `chromeTolerance` and it's a percent value from 0 to 100. Which is then divided by 100 into the code: https://github.com/oblador/loki/blob/8beda16fcda4841c4e23c7353880e94498cbadcc/packages/diff-pixelmatch/src/create-pixelmatch-differ.js#L61

My tests with pixelmatch object were not good. So I guess it changed somehow?

File: .lokirc [hunk 1]
```diff
@@ -1,9 +1,10 @@
 {
   "diffingEngine": "pixelmatch",
-  "chromeTolerance": 0.02,
+  "chromeTolerance": 1,
```
[the remark this answers is not in the document]
Comment (vincentaudebert) [reply .lokirc]: no reason, testing leftover, I moved back to 0,02 hoping it won't break anything. We will see

**Review by matthprost — APPROVED** [review]

**Pull request merged**